docker/pulsar/scripts/gen-yml-from-env.py

```diff
@@ -75,6 +75,19 @@
                 conf_to_modify = conf_to_modify[key_part]
                 modified = True
             i += 1
+
+    containerFactory=os.environ['PF_containerFactory']
```

Comment:
What about this `os.environ.get('PF_containerFactory', None)`? The current way will throw an exception if the key does not exist and the script will fail and not update the file.

Reply:
done